habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_policy/defined_skills/oracle_skills.yaml

```diff
@@ -60,7 +60,7 @@ nav_to_obj:
   apply_postconds: True
   force_end_on_timeout: False
   obs_skill_input_dim: 2
-  pddl_action_names: ["nav", "nav_to_receptacle_by_name"]
+  pddl_action_names: ["nav_to_obj", "nav_to_goal", "nav_to_receptacle_by_name"]
```

Comment:
Should hab3_planner.py use nav_to_obj and nav_to_goal?

Reply:
I don't think so because of above comment.